components/Layout/Footer.vue

```diff
@@ -36,7 +36,8 @@ export default defineNuxtComponent({
         >
       </a>
     </span>
-    <CookiesConsent />
+    <!-- TODO: Fix client-side error -->
+    <!-- <CookiesConsent /> -->
```

Comment:
It look like we have some server code executing on client-side from `config.ts:vidos()` as we are trying to read from local JSON file.